gpu/Makefile

```diff
@@ -85,4 +85,12 @@ depend:
 	    $(CXXCPP) $(CPPFLAGS) -x c++ -MM $$i; \
 	done > depend
 
+install: libgpufaiss.a libgpufaiss.$(SHAREDEXT) installdirs
+	cp libgpufaiss.a libgpufaiss.$(SHAREDEXT) $(DESTDIR)$(libdir)
+	cp *.h $(DESTDIR)$(includedir)/faiss/gpu
+	cp --parents **/**.h $(DESTDIR)$(includedir)/faiss/gpu
```

Comment:
The `--parents` option of `cd` is not standard but as this target will likely only run on Linux for now it's not a big deal.